wavelink/ext/spotify/__init__.py

```diff
@@ -197,6 +200,7 @@ def __init__(self, data: dict[str, Any]) -> None:
         self.length: int = data['duration_ms']
         self.duration: int = self.length
         self.isrc: str | None = data.get("external_ids", {}).get('irsc')
+        self.explicit: bool = True if data.get('explicit', False) == "true" or data.get('explicit', False) == True else False
```

Comment:
Could this not simply be  
`data.get('explicit', False) in {"true", True}`

Reply:
Yes this could be simplified like this, thanks